Assets/MRTK/SDK/Features/UX/Scripts/Utilities/PrefabSpawner.cs

```diff
@@ -245,5 +254,10 @@ private void HandleFocusExit()
         {
             focusExitTime = Time.unscaledTime;
         }
+
+        private Boolean HasVanishDelayElapsed()
+        {
+            return Time.time - focusExitTime > vanishDelay;
```

Comment:
Just want to double check that you intend to use Time.time vs Time.unscaledTime as the latter is used in several other places in the file.

Reply:
Good question! I copied this over from the existing part, but unscaledTime is right (i.e. focusExitTime is in regular time, which is bad).